internal/pkg/cli/env_init.go

```diff
@@ -13,22 +13,53 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/copilot-cli/internal/pkg/aws/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/ec2"
 	"github.com/aws/copilot-cli/internal/pkg/aws/identity"
 	"github.com/aws/copilot-cli/internal/pkg/aws/profile"
 	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
 	deploycfn "github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
 	"github.com/aws/copilot-cli/internal/pkg/term/color"
 	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
+	"github.com/aws/copilot-cli/internal/pkg/term/selector"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
+type customizedEnvTypeSelection int
+
+func (s customizedEnvTypeSelection) String() string {
+	return envInitCustomizedEnvTypes[s]
+}
+
 const (
-	envInitNamePrompt        = "What is your environment's name?"
-	envInitNameHelpPrompt    = "A unique identifier for an environment (e.g. dev, test, prod)."
-	envInitProfileHelpPrompt = "The AWS CLI named profile with the permissions to create an environment."
+	envInitImportEnvResources customizedEnvTypeSelection = iota
+	envInitAdjustEnvResources
+	envInitUseDefaultEnv
+)
+
+const (
+	envInitNamePrompt              = "What is your environment's name?"
+	envInitNameHelpPrompt          = "A unique identifier for an environment (e.g. dev, test, prod)."
+	envInitProfileHelpPrompt       = "The AWS CLI named profile with the permissions to create an environment."
+	envInitDefaultEnvConfirmPrompt = `Would you like to use the default configuration for a new production environment?
+    - A new VPC with 3 AZs, 2 public subnets and 2 private subnets
+    - A new ECS Cluster
+    - New IAM Roles
+    - Termination Protection for resources
+`
+	envInitConfigImportSelectPrompt   = "Ok no problem - would you like to adjust the environment config or import your own resources?"
+	envInitVPCSelectPrompt            = "Which VPC would you like to use?"
+	envInitPublicSubnetsSelectPrompt  = "Which public subnets would you like to use?"
+	envInitPrivateSubnetsSelectPrompt = "Which private subnets would you like to use?"
+
+	envInitVPCCIDRPrompt     = "What VPC CIDR would you like to use?"
+	envInitPublicCIDRPrompt  = "What CIDR would you like to use for the first public subnet?"
+	envInitPrivateCIDRPrompt = "What CIDR would you like to use for the second public subnet?"
```

Comment:
These variable names and prompt messages don't match 🤔

Reply:
🤦‍♂️